### PR TITLE
[5.1] Use static instead of self

### PR DESCRIPTION
**Pull request by lucasmichot**
Use static instead of self

### CODE REVIEW

File: src/Illuminate/Foundation/Console/IlluminateCaster.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public static function castApplication(Application $app)
     {
         $results = [];
 
-        foreach (self::$appProperties as $property) {
+        foreach (static::$appProperties as $property) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/IlluminateCaster.php]: appProperties is private

Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/IlluminateCaster.php]: revert this change please. the other one looks ok

**Comment by lucasmichot** [comment]
Squashed in https://github.com/lucasmichot/framework/commit/7fe79c5f43820ed64d1bed86ed6d709c7bf51e73 @GrahamCampbell

**Comment by GrahamCampbell** [comment]
:+1:
